config/config.exs

```diff
@@ -33,7 +33,9 @@ config :logger, :console,
   metadata: [:request_id]
 
 config :kafka_ex,
-  disable_default_worker: true
+  disable_default_worker: true,
+  use_ssl: false,
+  ssl_options: []
```

Comment:
Unrelated: this drives me nuts.  It causes so many problems.  I want `ssl_options` to die in a fire.